.github/templates/model_regression_test_config_to_json.tmpl

```diff
@@ -16,6 +16,11 @@ core
 {{- if has $config "dataset_branch" -}}
 {{- $dataset_branch = $config.dataset_branch -}}
 {{- end -}}
+{{- $num_repetitions := 1 -}}
+{{- /* if a branch name for dataset repository is not defined use the main branch */ -}}
```

Comment:
Not sure this comment is in the right place anymore, but shouldn't matter since you'll want to remove it entirely as a result of [this PR](https://github.com/RasaHQ/rasa/pull/10786).

Reply:
Good catch